.storybook/components/OrderSummaryStory.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { storiesOf, action } from '@storybook/react';
+import {
+  OrderSummary,
+  OrderSummaryHeader,
+  OrderSummaryCategory,
+  OrderSummaryList,
+  OrderSummaryListItem,
+  OrderSummaryTotal,
+  OrderSummaryFooter,
+} from '../../components/OrderSummary';
+import Button from '../../components/Button';
+import Dropdown from '../../components/Dropdown';
+import DropdownItem from '../../components/DropdownItem';
+
+storiesOf('OrderSummary', module)
+  .addWithInfo(
+    'Simple',
+    `
+      description here
```

Comment:
Add a real description in all of the stories.
If there's anything specific to the react component that should be noted, also include it in the description.